src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -117,6 +117,7 @@ public string ClientVersion
         public bool RetryOnServerErrors { get; set; } = true;
 
         public ManagedIdentityId ManagedIdentityId { get; internal set; }
+        public bool DisableInternalRetries { get; internal set; }

```

Comment:
This is fine, the name is better. But then, should you not delete `RetryOnServerErrors ` ?

Reply:
done